tubular/nominal.py

```diff
@@ -403,6 +425,12 @@ def transform(self, X):
 
         self.check_is_fitted(["mapping_"])
 
+        if not self.encode_unseen_levels:
+            for c in self.columns:
+                unseen_vals = set(X[c]) - set(self.training_data_levels[c])
+                for unseen_val in unseen_vals:
+                    self.mapping_[c].append(unseen_val)
```

Comment:
Predates this PR but on seeing it here and getting stuck for a moment in confusion I'm wondering if `mapping_` is a bit of a misleading variable name in this transformer?  Would `non_rare` be clearer and a quick fix?

Reply:
I agree - I went for non_rare_levels as I felt that was even clearer, but happy to go for non rare if you disagree. Done here:

https://github.com/lvgig/tubular/pull/141/commits/016a45b58fbaba7ea7aaab7a3b93305e80afe4f2